stacks/go/1.0.2/devfile.yaml

```diff
@@ -1,14 +1,14 @@
 schemaVersion: 2.1.0
 metadata:
-  description: "Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
+  name: go
   displayName: Go Runtime
+  description: Go (version 1.16) is an open source programming language that makes it easy to build simple, reliable, and efficient software.
```

Comment:
The image doesn't contain Go 1.16 but 1.18. 
I'm not sure if the `latest` tag of this image pins the Go version to any specific version. But there are tags that define specific Go versions

Reply:
Most probably my confusion here. I'm mentioning the version that is defined inside the `main.go` of the application